_maps/map_files/RandomRuins/SpaceRuins/wreckedcargoship.dmm

```diff
@@ -0,0 +1,2004 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"ae" = (
+/obj/machinery/light/small{
+	dir = 8
+	},
+/obj/structure/closet/walllocker/emerglocker{
+	dir = 8;
+	pixel_x = -32
+	},
+/obj/effect/decal/cleanable/dirt,
+/turf/simulated/floor/pod/light,
+/area/ruin/space/wreck_cargoship)
+"ah" = (
+/obj/effect/decal/warning_stripes/yellow,
+/obj/machinery/door/poddoor/shutters{
+	id_tag = cargoshipblastdoor2
```

Comment:
contain id_tags in double quotes e.g. `id_tag = "cargoshipblastdoor2"`

Reply:
oops